packages/deployment/lib/kes.js

```diff
@@ -286,8 +286,8 @@ class UpdatedKes extends Kes {
     );
 
     Handlebars.registerHelper(
-      'ifDeployDistribution', (configs, apiUrl, options) =>
-        ((Object.keys(configs).includes('CumulusApiDistribution') && !apiUrl)
+      'ifDeployApi', (templateKey, deployDistribution, options) =>
+        ((templateKey !== 'CumulusApiDistribution' || deployDistribution)
```

Comment:
I may not be getting the full scope of this helper, but would it be easier to just do `(templateKey == 'CumulusApiDistribution' && deployDistribution)`?

The way it's currently written, this helper evaluates to true anytime it's called with a `templateKey` of `not 'CumulusApiDistribution'`. Is that the intended behavior?

Reply:
yes